src/library_html5.js

```diff
@@ -2759,6 +2759,28 @@ var LibraryJSEvents = {
     return !GL.contexts[target] || GL.contexts[target].GLctx.isContextLost(); // No context ~> lost context.
   },
 
+#if USE_WEBGPU
+  emscripten_webgpu_get_device__sig: 'i',
+  emscripten_webgpu_get_device: 'emscripten_webgpu_do_get_device',
+  emscripten_webgpu_do_get_device__deps: ['$WebGPU'],
+  emscripten_webgpu_do_get_device: function() {
```

Comment:
The reason why there is this kind of dual scheme `emscripten_webgpu_get_device` and `emscripten_webgpu_do_get_device` on some of the other functions is due to pthreads proxying that is dictated at runtime. That is, we don't know for those functions at compile time whether they need to be proxied, but the decision is made at runtime. Currently there is no multithreaded proxying present in these two functions, so you can just delete the `xxx_do_xxx` variants and have the function named as `emscripten_webgpu_get_device` (and likewise for (`emscripten_webgpu_get_current_texture`) variants.

Once/if runtime dictated proxying comes into play in future, then it can be refactored under the hood to do the proxying without needing to break public visible API, so it's a safe change to leave for later if it's ever needed.